### PR TITLE
Revamp togglebox

### PR DESCRIPTION
**Pull request by osedoe**

### CODE REVIEW

File: src/components/boxes/ToggleBox.tsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-import * as styles from './ToggleBox.css';
 import styled from '@emotion/styled';
+import { Checkbox } from '../..';
 
 export interface ToggleBoxProps {
   title: string;
+  subtitle?: string;
   info?: string;
-  type: 'danger' | 'normal';
+  type?: 'danger' | 'normal';
```
Comment (raulanatol) [inline src/components/boxes/ToggleBox.tsx]: Remove this type ;)

File: src/components/boxes/ToggleBox.tsx [hunk 2]
```diff
@@ -19,37 +18,54 @@ export interface ToggleBoxState {
...
+  return {
+    background: '#4A90E2',
+    checkbox: {
+      bulletColor: '#4A90E2'
+    }
+  };
 }
 
 export class ToggleBox extends React.Component<ToggleBoxProps, ToggleBoxState> {
+  static defaultProps: Partial<ToggleBoxProps> = {
```
Comment (raulanatol) [inline src/components/boxes/ToggleBox.tsx]: Is it possible to refactor to a functional component?

File: src/components/boxes/ToggleBox.tsx (hunk 1 above)
Comment (osedoe) [reply src/components/boxes/ToggleBox.tsx]: Done!

File: src/components/boxes/ToggleBox.tsx (hunk 2 above)
Comment (osedoe) [reply src/components/boxes/ToggleBox.tsx]: Sure. I'll give it a go :)

File: src/components/boxes/ToggleBox.tsx [hunk 3]
```diff
@@ -46,48 +40,40 @@ function getTheme(toggleChecked: boolean) {
...
+export const ToggleBox = (props: ToggleBoxProps) => {
+  const {
+    title,
+    info,
+    value = true,
+    name,
+    disabled,
+    onChange
+  } = props;
+  const [stateValue, setValue] = useState(!!value);
```
Comment (raulanatol) [inline src/components/boxes/ToggleBox.tsx]:
```
const [value, setValue] = useState(!!props.value);
```

File: src/components/boxes/ToggleBox.tsx [hunk 4]
```diff
@@ -46,48 +40,40 @@ function getTheme(toggleChecked: boolean) {
...
-  constructor(props: ToggleBoxProps) {
-    super(props);
-    this.state = { value: !!props.value };
-    this.toggle = this.toggle.bind(this);
-  }
+export const ToggleBox = (props: ToggleBoxProps) => {
+  const {
+    title,
+    info,
+    value = true,
```
Comment (raulanatol) [inline src/components/boxes/ToggleBox.tsx]: Remove this line

**Pull request merged**